qt/main.cpp

```diff
@@ -215,20 +214,19 @@ int main(int argc, char * argv[])
     fmt.setSwapInterval(1);
     fmt.setDepthBufferSize(16);
 
-    // This is a workaround for some systems,
-    // including MacOs and older Linux distros, which rely on X.org and Mesa.
-    // Where somehow the driver itself doesn't
-    // make all the otherwise supported GLSL versions available by default
-    // and such requests are somehow disregarded at later stages of execution.
-    // This setting here will be potentially overwritten and overruled anyway,
-    // and only needed to ensure that we have the needed GLSL compiler available
-    // later when we need it.
-    if (app.platformName() == QString("xcb") ||
-        app.platformName() == QString("cocoa"))
+    if (app.platformName() == QString("wayland"))
```

Comment:
Why do we need this branch? Everything worked perfectly fine on the wayland `platform` without it. I suggest [Occam's razor](https://en.wikipedia.org/wiki/Occam%27s_razor). I think it is the`else` branch which delivers value.

Reply:
So you propose to keep only this?
https://github.com/organicmaps/organicmaps/pull/5014#issuecomment-1518933592

Reply:
Exactly.